app/models/BasicPokemonDetail.js

```diff
@@ -11,14 +12,7 @@ let BasicPokemonDetailSchema = new mongoose.Schema({
         type: String
     },
     gender: {
-        /*male: {
-            type: Boolean
-        },
-        female: {
-            type: Boolean
-        }*/
-        male: Boolean
-
+        type: String  //only use of values 'm' for male, 'f' for female, 'u' for undefined and 'h'
```

Comment:
There is a way to limit entries in a mognoose schema!! ;)

http://mongoosejs.com/docs/api.html#schema_string_SchemaString-enum

And I suggest you also normalize input by http://mongoosejs.com/docs/api.html#schema_string_SchemaString-lowercase

I don't though know the sequence in enum + toLowerCase (meaning, if it will first apply the lower case to, for example ,`M` and then validate it (which will be ok), or the other way around. The first way would work, the second way would produce an error. Test it! :smile: ).

Third option is write a validating function:
http://mongoosejs.com/docs/api.html#schematype_SchemaType-validate